test/component/loglevel_test.js

```diff
@@ -61,8 +61,28 @@ describe('LogLevel', function() {
                 done();
             });
         });
+    });
+    describe('#GetLoglevel()', function() {
+        it('should return the current log level', function(done) {
+            var levels = ['FATAL', 'ERROR', 'WARN', 'WARNING', 'INFO', 'DEBUG'];
```

Comment:
WARNING is no longer a supported alias for WARN (i.e. the token to use for warning level is WARN). Thus, I understand it would be removed from the `levels` array.

Apart from this test, maybe the "removal" of WARNING have additional impact in some other parts of the code.